client/components/event/RegisterEventCard/RegisterEventCard.tsx

```diff
@@ -153,7 +172,8 @@ function EventCard({
                 <span className="label text-bold mb-2">
                   Nume si prenume
                 </span>
-                <input type="text" name="name" required />
+                {userInfo && <input type="text" value={username} name="name" required onChange={onUsernameChange} />}
```

Comment:
Let's not add local state for username & email. If we use `defaultValue` prop on inputs it should be enough. And we can get rid of setStates and onUsernameChange/onEmailChange.